app/javascript/react/components/ReviewTile.js

```diff
@@ -1,15 +1,19 @@
-import React from "react";
+import React from 'react';
 
 const ReviewTile = (props) => {
   let coffeeRating = [];
   for (let i = 0; i < props.rating; i++) {
     coffeeRating.push(<i className="fa fa-coffee" aria-hidden="true"></i>);
   }
-  let deleteButton
-  if (props.currentUser.role === "admin") {
-    deleteButton = <button type="button" className="alert button" onClick={props.handleClick}>Delete</button>
+  let deleteButton;
+  if (props.currentUser?.role === 'admin') {
+    deleteButton = (
+      <button type="button" className="alert button" onClick={props.handleClick}>
+        Delete
+      </button>
+    );
   } else {
-    deleteButton = null
+    deleteButton = null;
   }
```

Comment:
this is fine, but you can actually skip the `else` statement. Just on line 8 set `deleteButton` to `null` by default.